rpc/src/rpc_impl.rs

```diff
@@ -493,8 +493,10 @@ impl RpcFunctions for RpcImpl {
             })
             .collect();
 
-        let potential_forks = known_network
-            .potential_forks()
+        let (potential_tip, potential_forks) = known_network.potential_forks();
+        let potential_tip = potential_tip.map(|(height, members)| PotentialFork { height, members });
```

Comment:
maybe `PotentialFork` could have its name changed to `NodeCluster`, now that it applies to both forks and the possible canon cluster?